docs/docs/naming.md

```diff
@@ -44,9 +74,9 @@ $ dig _search._tcp.marathon.mesos SRV
 
 ;; ANSWER SECTION:
 _search._tcp.marathon.mesos.	60 IN SRV 0 0 31302 10.254.132.41.
-``` 
+```
 
-SRV records are generated only for tasks that have been allocated a specific port through Mesos. 
+*Note*: SRV records are generated only for tasks that have been allocated a specific port through Mesos and that have no container IP assigned.
```

Comment:
But for container ip SRV records we have to know whether the discoveryinfo really means the container ports, not the host ports. So a framework needs knowledge about the infrastructure to publish the right ports.